ci/taos/plugins-base/pr-postbuild-build-tizen.sh

```diff
@@ -66,13 +66,15 @@ function pr-postbuild-build-tizen-run-queue(){
     # Build a package with gbs command.
     # TODO: Simplify the existing if...else statement for readability and maintenance
     echo -e "[DEBUG] gbs build start at : $(date -R)."
+    _TIZEN_GBS_PROFILE="-P ${TIZEN_GBS_PROFILE}"
     if [[ $BUILD_MODE == 99 ]]; then
         echo -e "BUILD_MODE = 99"
         echo -e "Skipping 'gbs build -A $1' procedure temporarily."
     elif [[ $BUILD_MODE == 1 ]]; then
         echo -e "BUILD_MODE = 1"
         sudo -Hu www-data gbs build \
         -A $1 \
+        ${_TIZEN_GBS_PROFILE} \
```

Comment:
It seems that only `-P` might be passed when `TIZEN_GBS_PROFILE` is empty. Is it okay?

Reply:
Thanks. Fixed.